practice-app/backend/controllers/covidNews.js

```diff
@@ -0,0 +1,63 @@
+const Response = require('../utils/response');
+const Request = require('../utils/request');
+
+/**
+ * Explanation
+ * Response: {
+ *    status: {
+ *        success: true,
+ *        code: 200
+ *    }, data: {
+ *        articles: [
+ *            {
+ *                title: 'Headline for the first article',
+ *                description: 'A short description of this article',
+ *                url: 'https://www.example.org'
+ *            },             
+ *            {
+ *                title: 'Headline for the second article',
+ *                description: 'A short description of this article',
+ *                url: 'https://www.example2.org'
+ *            },
+ *            ... // the number of articles varies, max num = 10
+ *        ]
+ *    }
+ *}
+ */
+module.exports.getNews = async function (request, response) {
+    let current_date = new Date().toISOString();
+    current_date = current_date.substring(0, current_date.indexOf('T'));
+
+    try {
+        Request.send({
+            url: 'https://newsapi.org/v2/top-headlines?q=COVID&from=' + current_date + '&sortBy=publishedAt&apiKey=' + process.env.COVID_NEWS_API_KEY + '&pageSize=10&page=1&country=tr',
```

Comment:
Maybe "koronavirüs" is a more used term in Turkey than "COVID". It can be changed in the future if we won't have enough news to show in the website related to the coronavirus.

Reply:
You are right. Thanks for the suggestion. I have changed my code as you suggested.

Reply:
Saw that just now :D No worries!